app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, "About") %> 
+<p>
+  The <a href="https://www.railstutorial.org/"><em>Ruby on Rails Tutorial</em></a>, part of the
+  <a href="https://www.learnenough.com/">Learn Enough</a> family of tutorials, is a
```

Comment:
dùng helper link to thay cho thẻ a

Reply:
trong tutorial chương 3 vẫn dùng <a></a> chị ạ